igrill.py

```diff
@@ -110,6 +113,9 @@ def authenticate(self):
 
     def read_battery(self):
         return float(bytearray(self.battery_char.read())[0])
+    
+    def read_heating_elements(self):
+        return bytearray(self.heating_elements.read())
```

Comment:
Is the way you interpret the output of the heating element different from the other temperature probes?